### DISCUSSION TITLE
Is Connected?

### DISCUSSION THREAD

**Discussion by pneumus** [hbldh/bleak#645] [Ideas]
[Feature Request]

On Manjaro Linux I was able to add an is_connected method:

    @staticmethod
    def is_connected():
        """ Return a boolean value if the device is connected """
        cmd = "bluetoothctl info | grep Connected | awk '{print $2}'"
        connected = subprocess.check_output(cmd, shell=True)
        if connected == b'yes\n':
            return True
        else:
            return False

A similar method shall be implemented on Win and Mac.
Anybody wrote these already?
Thanks!

**Comment by dlech** [comment]
There is already a `BleakClient.is_connected` property. How is this different?
